### ISSUE TITLE
Set certificate for OpenID provider?

### ISSUE THREAD

**Issue by azurewraith** [int128/kubelogin#5]
I didn't see it in the documentation, is there a way to override the OpenID client side certificate?

```
$ kubelogin
2018/08/11 03:17:22 Reading /home/azurewraith/.kube/config
2018/08/11 03:17:22 Using current context: stage
2018/08/11 03:17:22 Authentication error: Could not access OIDC issuer: Get https://gitlab.domain/oauth/authorize/.well-known/openid-configuration: x509: certificate signed by unknown authority
```

I've tried `idp-certificate-authority` and `idp-certificate-authority-data` mentioned [here](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#using-kubectl) to no avail.

**Comment by int128** [comment]
I have released 1.2 with `--insecure-skip-tls-verify` support.

`idp-certificate-authority` and `idp-certificate-authority-data` will be supported in the future.

**Comment by int128** [comment]
Please try the new release.
